hack/drone.jsonnet

```diff
@@ -426,7 +425,6 @@ local release = {
       '_out/installer.tar',
       '_out/osctl-darwin-amd64',
       '_out/osctl-linux-amd64',
-      '_out/talos.iso',
```

Comment:
I think we should keep everything else. This line should be enough to not publish it.

Reply:
k - amended.